plugins/ScheduledReports/API.php

```diff
@@ -122,6 +122,10 @@ public function addReport(
 
         self::validateCommonReportAttributes($period, $hour, $description, $idSegment, $reportType, $reportFormat, $evolutionPeriodFor, $evolutionPeriodN);
 
+        if (null !== $periodParam) {
```

Comment:
Are we adopting yoda style? :)

Strictly technically speaking it is safer and can be checked by a linter...

Reply:
I prefer using yoda style, even though I sometimes miss to use it myself 🙈 
But at some point we can consider to force that using a linter maybe.